scripts/debian/build.sh

```diff
@@ -37,8 +37,7 @@ if [ $# -eq 0 ]
 
   else
     for i in "$@"; do
-      if [[ $(type -t "build_${i}_deb") == function ]]
-      then
+      if declare -f "build_${i}_deb" > /dev/null; then
```

Comment:
This would be less efficient, as we're printing the definition, although to `/dev/null`